src/app.ts

```diff
@@ -1,6 +1,87 @@
 
 const startApp = async () => {
+    const tbody = document.querySelector('[data-sink]')
+    const tr = tbody?.querySelectorAll('tr')
+    const prevBtn = document.querySelector('[data-prevbtn')
+    const nextBtn = document.querySelector('[data-nextbtn')
+    let url = 'https://randomapi.com/api/8csrgnjw?key=LEIX-GF3O-AG7I-6J84&page=1'
+    let response = await fetch(url)
+    let currentIndex = 1
+    let dataStore = {}
+    if(response.ok){
```

Comment:
Codacy has a fix for the issue: [Replace `··if(response.ok)` with `if·(response.ok)·`](https://app.codacy.com/gh/alic-xc/Test/pullRequest?prid=9965044)
```suggestion
  if (response.ok) {
```